include/experimental/group/gemm/impl/int4_dequantize_xmx_xe.hpp

```diff
@@ -353,9 +359,9 @@ class gemm_t<
         xetla_nbarrier_t<wg_size_x, wg_size_x, arch_tag> nbarrier_a;
         nbarrier_a.init_nbarrier(
                 sg_idy + nbarrier_base, nbarrier_role::producer_consumer);
-        xetla_nbarrier_t<wg_size_y, wg_size_y, arch_tag> nbarrier_b;
-        nbarrier_b.init_nbarrier(sg_idx + barrier_count_y + nbarrier_base,
-                nbarrier_role::producer_consumer);
+        // xetla_nbarrier_t<wg_size_y, wg_size_y, arch_tag> nbarrier_b;
+        // nbarrier_b.init_nbarrier(sg_idx + barrier_count_y + nbarrier_base,
+        //         nbarrier_role::producer_consumer);
```

Comment:
shall there be nbarrier?

Reply:
There is only split_barrier in Dg2. Only one is used here.
I changed it to decide one or two based by gpu_arch.